test/pyrostest/CMakeLists.txt

```diff
@@ -0,0 +1,187 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(pyrostest)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+  std_msgs
+)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
```

Comment:
I don't think we should have all these boilerplate comments creating noise. Maybe have a comment that links to where one can find this file with these comments